stdlib/LibGit2/src/types.jl

```diff
@@ -198,29 +198,29 @@ Transfer progress information used by the `transfer_progress` remote callback.
 Matches the [`git_transfer_progress`](https://libgit2.github.com/libgit2/#HEAD/type/git_transfer_progress) struct.
 """
 @kwdef struct TransferProgress
-    total_objects::Cuint
-    indexed_objects::Cuint
-    received_objects::Cuint
-    local_objects::Cuint
-    total_deltas::Cuint
-    indexed_deltas::Cuint
-    received_bytes::Csize_t
+    total_objects::Cuint    = Cuint(0)
+    indexed_objects::Cuint  = Cuint(0)
```

Comment:
This should work if you just do `= 0`, right?

Reply:
It didn't until I removed the assertion that Jeff mentioned, so it does work now. I'd rather not go back through and change all of these to be less explicit though.